docs/samples/Microsoft.ML.AutoML.Samples/Microsoft.ML.AutoML.Samples.csproj

```diff
@@ -2,7 +2,8 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <CopyLocalLockFileAssemblies>false</CopyLocalLockFileAssemblies>
```

Comment:
Why is CopyLocalLockFileAssemblies=false added to these samples? Do they still run?

Reply:
Yeah they still run. Before when they were .netcore2 they didn't copy the nuget runtime folder into the output directory. I'm guessing the default changed with .netcore3 as the runtime folders are now copied into the output directory. This was causing issues during the CI process as it increased the space needed by like 3GB and that put us over the limit on the machines. This is preventing that copy from happening.